server_addon/tvpaint/server/settings/publish_plugins.py

```diff
@@ -1,5 +1,5 @@
 from ayon_server.settings import BaseSettingsModel, SettingsField
-from ayon_server.types import ColorRGBA_uint8, ColorRGB_uint8
+from ayon_server.types import ColorRGBA_uint8
```

Comment:
While we're at it. Should we also remove the commented lines on lines 16-18 along with this?

Reply:
The lines are actually correct and should be used. But that would require to implement conversion of settings in server addon.